lib/rules/no-fallthrough.js

```diff
@@ -55,6 +55,8 @@ function hasBlankLinesBetween(node, token) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I'd consider `no-fallthrough` to be a `problem`, since a fallthrough is often a bug.

Reply:
Agreed.

Reply:
I'm not sure I agree, for similar reasons to those I had laid out in `default-case`: If the rule is allowing the user to declare intent, then this is a suggestion, not a problem.

That said, fallthrough is usually a higher-impact issue than lack of default, so I could be okay with "problem" here.

Reply:
I agree that this should probably be the same as `default-case`, though I could see going either way on it.

Reply:
Setting to "problem".